tom/dependencies.py

```diff
@@ -177,7 +177,8 @@ def run(self, branch):
         """Run the dependency update for a branch, creating PR in the end"""
         self.slack.reply("Running dependency updates for " + branch)
         # prepare repo
-        self.buildscripts = GitRepo('../buildscripts', 'git@github.com:cfengine/buildscripts.git')
+        self.buildscripts = GitRepo(
+            '../buildscripts', 'git@github.com:cfengine/buildscripts.git', "cf-bottom")
```

Comment:
I'd prefer putting the first two args on the same line as the constructor call, but YMMV.